test/JDBC/expected/cast_eliminate-vu-verify.out

```diff
@@ -0,0 +1,82 @@
+set babelfish_showplan_all on
+GO
+
+SELECT 1 AS [C1] FROM cast_eliminate  WHERE (CAST(ROID AS BIGINT) = 1)
+GO
+~~START~~
+text
+Query Text: SELECT 1 AS [C1] FROM cast_eliminate  WHERE (CAST(ROID AS BIGINT) = 1)
+Index Only Scan using cast_eliminate_pkey on cast_eliminate  (cost=0.15..8.17 rows=1 width=4)
```

Comment:
Let's remove costs from output because it can vary across different versions. Ref: https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/working-with-babelfish-usage-notes-features.using.explain.html